charts/flyte-core/values.yaml

```diff
@@ -853,6 +853,28 @@ configmap:
       - id: production
         name: production
 
+  # -- Open Telemetry Configuration
+  otel:
+    otel:
+      # -- The type of exporter to use
+      type: noop
+      # -- Configuration for the file exporter type
+      file: "/tmp/trace.txt"
```

Comment:
@Sovietaced could it be that we are missing a `filename` here, see [this](https://github.com/flyteorg/flyte/blob/234de1fac735464035708f739341c0e08b1064aa/flytestdlib/otelutils/config.go#L64C25-L64C33)?

When upgrading from 1.16.0-b4 to 1.16.0 I see flytepropeller and flyte-pod-webhook crash on startup with this error:

```console
time="2025-09-17T11:12:46Z" level=error msg="\n\n1 error(s) decoding:\n\n* 'file' expected a map, got 'string'"
```